docs/.vuepress/sidebar/en.js

```diff
@@ -124,6 +124,12 @@ module.exports = [
       "plugin-development-manual",
       "user-defined-function",
       "spark-doris-connector",
+      "contribute_udf",
+      {
+        title: "Third-party UDF",
```

Comment:
```suggestion
        title: "Custom UDF",
```
Make it unify with `build_custom_udf.sh`